Create ui-c8y-1019-7-14-availability-translation-separated-for-jp-5755-graft-release-cd-5797.md (#1802)
SoftwareAG/c8y-docs: content/change-logs/ui-c8y-1019-7-14-availability-translation-separated-for-jp-5755-graft-release-cd-5797.md

---
date: ""

> title: Separate "Availability" translation for Japan

Reviewer: ```suggestion
title: Separate translation for the term "Aavilability" for Japanese
```

> title: "Availability" translation separated for JP (#5755) [GRAFT][release/cd] (#5797)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Separate "Availability" translation for Japan
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-57688
version: 1019.7.14
---

> "Availability" translation separated for JP (#5755) [GRAFT][release/cd] (#5797)

Author: The Claude AI suggested to use the following body:
```suggestion
The "Availability" text is now translated separately for Japan to improve localization. This allows the availability text to be tailored for the Japanese market.
```

> The "Availability" text is now translated separately for Japan to improve localization. This allows the availability text to be tailored for the Japanese market.

Reviewer: @janhommes What does that mean? The term "Availability"? Or what is meant by text? And what means "separately" here? Don´t get it.

Reviewer 2: @BeateRixen according to ticket description https://cumulocity.atlassian.net/browse/MTM-57688 in japanese language they have different word for 'availability' when it's about availability of dashboards (property indicating which role should current user have to edit particular dashboard) and availability of application package (based on package app type).

> The "Availability" text is now translated separately for Japan to improve localization. This allows the availability text to be tailored for the Japanese market.

Reviewer: ```suggestion
The term "Availability" in the UI now offers different translations for Japanese depending on the context. This improvement better tailors the localization for the Japanese market.
```
